templates/compose/pi-hole.yaml

```diff
@@ -0,0 +1,15 @@
+services:
```

Comment:
You forgot to add the metadata that is mentioned here: https://coolify.io/docs/knowledge-base/add-a-service#metadata

The port part will tell Coolify which port to use when configuring the proxy. Then you can use the env 
 variable `SERVICE_FQDN_PIHOLE`. Notice that you don't need the suffix `_80` anymore.
 
If you want to test your service you can do it in the dev mode mentioned here: https://github.com/coollabsio/coolify/blob/main/CONTRIBUTION.md#1-setup-your-development-environment
 
When the Coolify is running you can generate a new service-templates.json. More info here: https://github.com/coollabsio/coolify/issues/2375#issuecomment-2198225964

Reply:
I just tested this locally. And the traefik proxy is still using port 53 when trying to reach the WEB UI.
This is because traefik defaults to automatically proxy to exposed ports and Coolify doesn't override this.

I even tried using the postfix `_80` but it did not help.

I also tried to remove the `ports:` part from the docker-compose file. But traefik still sees the exposed ports that are hard coded into the original docker image. I.e. traefik will proxy to port 53.

I have to add labels manually to configure traefik. But that requires me to use the generated UID.

```yaml
labels:
      - traefik.http.routers.http-0-t4wgc4w-pihole.service=pi-hole
      - traefik.http.services.pi-hole.loadbalancer.server.port=80
```

I thought I knew how Coolify did its magic. But it seems that I was wrong :(